docker/nginx/nginx.conf

```diff
@@ -112,6 +112,12 @@ http {
             proxy_set_header Host $host;
         }
 
+	location /static {
+            alias /app/superset-frontend/src;
```

Comment:
Mmmmh, confused to see this as it seems to me the build directory for webpacked assets is `./superset/static/assets`
https://github.com/apache/superset/blob/master/superset-frontend/webpack.config.js#L43

`superset-frontend/src` contains some of the source code